README.md

```diff
@@ -202,6 +202,37 @@ MainScope mainScope = new MainScopeImpl();
 ChildScope childScope = mainScope.child();
 ```
 
+## Dependency Resolution and Exposure
+
+Motif requires explicit dependency declarations to properly resolve and inject dependencies across scopes. When a parent scope needs to provide a dependency to a child scope, the dependency must be explicitly exposed using the `@Expose` annotation.
+
+For example:
+
+```kotlin
+@motif.Scope
+interface ParentScope {
+    @motif.Objects
+    class Objects {
+        fun provideImpl(): Impl {
+            return Impl()
+        }
+
+        @Expose
+        fun provideApi(impl: Impl): Api {
+            return impl
+        }
+    }
+}
+
+@motif.Scope
+interface ChildScope {
+    @motif.Objects
+    abstract class Objects {
+        abstract fun api(): Api
```

Comment:
I'm not sure this is correct -- or, if it is, if it is needed. I think `Api` is already available in this scope because parent exposed it.

Reply:
Ah yeah you are right! It's not needed